chef/cookbooks/barclamp/libraries/barclamp_library.rb

```diff
@@ -160,6 +161,11 @@ def self.claimed(node,owner)
           end
         end
 
+        def self.multipath?(device = @device)
```

Comment:
This looks suspicious to me.  Where would `@device` come from when invoked as a class method?  Please could you add unit tests for this method, both with and without a parameter passed to it?  That would ensure it works in both cases.